js/graph_config.js

```diff
@@ -261,10 +261,11 @@ GraphConfig.load = function(config) {
         try {
             if (fieldName.match(/^motor\[/)) {
                 return {
-                    offset: -(sysConfig.motorOutput[1] + sysConfig.motorOutput[0]) / 2,
+                    offset: sysConfig.fast_pwm_protocol < FAST_PROTOCOL.indexOf('DSHOT150') ?
```

Comment:
Again, this should use the actual values for the range.

Reply:
What actual values should be used?

Reply:
The ones for the range - or what others can you think of?

Reply:
I'm not intending to be obtuse to annoy.
I really do not know what range you are referring to.
You could mean the lowest or highest value received in the log for all I know.